### ISSUE TITLE
"Clear filters" button should be disabled until there are filters to clear

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4086]
## Area or Component
Results page - filter component

## Quick Summary
The "clear filters" button on the results page is enabled even when there are no filters to clear.

## Action Requested
Make the button disabled until filters have been applied. 
Please ensure that the button is disabled both visually and for screenreader users.

## Screenshots (optional)
![Screen Shot 2022-07-26 at 4 26 07 PM](https://user-images.githubusercontent.com/80282552/181129065-c4f2f54b-93d3-46a7-8789-48bac47f545e.png)
